overops-functions/src/main/java/com/takipi/udf/microsoftteams/card/MicrosoftActivitySection.java

```diff
@@ -0,0 +1,78 @@
+package com.takipi.udf.microsoftteams.card;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MicrosoftActivitySection implements MicrosoftSection {
+    public String activityTitle = "";
```

Comment:
Members of this class can probably be `final`

Reply:
done